scalableminds/webknossos
# Auth dev

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
This PR replaces the Basic Authentication (that caused trouble with volume tracings) with the authentication integrated into webKnossos.

### Steps to test:
- deploy branch on dev
- Basic Authentication should be disabled
- auto-login should be disabled for scmboy

### Issues:
- fixes #1919

------
